docs/source/tutorial/project_setup.rst

```diff
@@ -0,0 +1,42 @@
+==============
+Project Setup
+==============
+
+If you haven't done so already make sure to complete the installation before moving on.
+
+Create Your Ingest Project
+==========================
+
+Create a new ingest project named ``My_Study`` by using the ingest library CLI::
+
+$ kidsfirst new My_Study
+
+This will generate a minimal study directory for you. It will look like this::
+
+    My_Study/
+    ├── dataset_ingest_config.yml
+    ├── extract_configs/
+    │   ├── example_extract_config.py
+    └── transform_module.py
+
+If you would like, you can fill in the study and investigator information in the
+``dataset_ingest_config.yml`` file. We won't actually use this until later on in the load
+stage so you can always come back to modify it. Leave everything else as is for now.
+
+Get Some Source Data
```

Comment:
I think we should either have a special CLI command for generating the full tutorial ingest project or have them copy over the data files from somewhere. I like having them do it manually because its closer to what they might do with local data for a study ingest of their own.

Reply:
👌